gradle.properties

```diff
@@ -1,3 +1,22 @@
+#
+#    Transportr
+#
+#    Copyright (c) 2013 - 2021 Torsten Grote
+#
+#    This program is Free Software: you can redistribute it and/or modify
+#    it under the terms of the GNU General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
```

Comment:
Not sure we want this and the two ones below. Maybe exclude those files in Android Studio's copyright wizard?

Reply:
Okay, no problem. I'll update it.